LICENSE

```diff
@@ -200,3 +200,19 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
```

Comment:
Got more comment on license. 

The license of theta sketches has  "4. Redistribution" section details many of the obligations for copies. (https://github.com/apache/incubator-datasketches-cpp/blob/master/LICENSE)

Please take a look and double check if we have carried over as guided there. 

When in doubt, let's shoot an email to opensource-licensing@google.com